yutuf_serch.py

```diff
@@ -0,0 +1,27 @@
+import requests
+from googleapiclient.discovery import build
+
+def get_youtube_video_ids(api_key, search_query):
+    youtube = build('youtube', 'v3', developerKey=api_key)
+
+    search_response = youtube.search().list(
```

Comment:
Here can be added a filter for input , just in case other videos unrelated to the query will pop up in the id results